docs/swg-basic-docs/enable_swg.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Enable your publication to use Subscribe with Google subscriptions or contributions.
+---
+
+# Enable Subscriptions or Contributions
+
+In order to enable Subscribe with Google for your publication, each page on your site must be configured with a code snippet generated by Publisher Center. This snippet loads the Subscribe and Contribute features, it also associates your pages with one or many pricing plans. To install the snippet, you must have permission to access and edit your site’s theme or template code in your content management system (CMS).
+
+## Add code snippet to your site
+
+- Open [Publisher Center](https://publishercenter.google.com)
+- Select your publication with a Subscriptions or Contributions plan enabled. Make sure you have a live pricing plan.
+- Select the “Subscribe with Google” panel
+- Under Quick Links, select CMS Sync
+- Navigate to the CMS Sync Tab in Publisher Center. You will find:
+  - Open Access Snippet, which gives readers open access to all content
+  - Product ID Snippet, which you can use for paywalled content
+- Add the snippet to the `<head>` tag of each page that you want to enable Subscribe with Google
+- Navigate to your website to ensure Subscribe with Google is loading
+
+
+Add this code snippet into your content management system to give readers access. Specify your `PRODUCT_ID` and which offer you want to display in the  `isPartOfProductId`. For Contributions, your `autoPromptType` should be set to `contribution`, and for Subscriptions, it should be set to `subscription`.
+
+```javascript
+<script async type="application/javascript"
+        src="https://news.google.com/swg/js/v1/swg-basic.js"></script>
+<script>
+  (self.SWG_BASIC = self.SWG_BASIC || []).push( basicSubscriptions => {
+    basicSubscriptions.init({
+      type: "NewsArticle",
+      isAccessibleForFree: false,
+      isPartOfType: ["Product"],
+      isPartOfProductId: "<PRODUCT_ID>:basic",
+      autoPromptType: "contribution",
+      clientOptions: { theme: "light", lang: "en" },
+    });
+  });
+</script>
+```
+
+> **Note:** If you make a change to your pricing with a new product ID, you’ll need to update your CMS Sync on your website. Also, if you will have multiple plans active on a single site (for example, some pages always free and some pages paid), you will need to configure multiple snippets, one for each product ID, and make sure to include them on the relevant pages and/or templates within your CMS.
+
+## Code snippet properties
+
+`type`
```

Comment:
Not sure if you were going for line breaks, but it needs a double space at the end of each line
```diff
-`type`
-`isAccessibleForFree`
+`type`  
+`isAccessibleForFree`  
```
<img width="602" alt="image" src="https://user-images.githubusercontent.com/1211229/171418535-4b5b4f40-f19c-4517-89e3-9c377c0c75ee.png">

vs

<img width="279" alt="image" src="https://user-images.githubusercontent.com/1211229/171418626-4c829639-3865-439f-805c-c64fd540c570.png">